lib/web_console/railtie.rb

```diff
@@ -10,7 +10,9 @@ class Railtie < ::Rails::Railtie
       require "bindex"
       require "web_console/extensions"
 
-      ActionDispatch::DebugExceptions.register_interceptor(Interceptor)
+      ActiveSupport.on_load(:action_dispatch_request) do
+        ActionDispatch::DebugExceptions.register_interceptor(Interceptor)
```

Comment:
`register_interceptor` is public API and has other consumers, e.g:

https://github.com/mattes/google_cloud_run/blob/2c67e17e5716ee0ebe3e7cae8c2237ea0fac5010/lib/google_cloud_run/railtie.rb#L39
https://github.com/discourse/rails_failover/blob/1698a6f92b6827366f5cf172db5ec43fe0d8c81e/lib/rails_failover/active_record/railtie.rb#L55

It shouldn't be necessary to wrap it in a load hook like this to avoid breaking the host application.

Reply:
Ok makes sense 😄 . I'll close this. Thanks!